### PR TITLE
ISSUE-43071: Fixing some language issues

### LINKED ISSUE #43071: [enterprise-4.10] Issue in file updating/preparing-eus-eus-upgrade.adoc
### Which section(s) is the issue in?
https://docs.openshift.com/container-platform/4.10/updating/preparing-eus-eus-upgrade.html#updating-eus-to-eus-upgrade_eus-to-eus-upgrade

### What needs fixing?

1. This language is confusing. Turn the negative into positive.
`Verify that your cluster is not running a version earlier than OpenShift Container Platform 4.8.14. 
`
should be:
`Verify that your cluster is running OpenShift Container Platform version 4.8.14 or later.
`

2. Seem to be missing a word here:
`If your cluster is running a version` **earlier** `than OpenShift Container Platform 4.8.14, you must update to a later 4.8.z version prior to updating to 4.9. 
`

### PR DESCRIPTION
**Pull request by sagidlow**
Applies to 4.8+ 
GH Issue: https://github.com/openshift/openshift-docs/issues/43071
Minor change, QE not required.
Preview Link: https://deploy-preview-43851--osdocs.netlify.app/openshift-enterprise/latest/updating/preparing-eus-eus-upgrade.html#updating-eus-to-eus-upgrade_eus-to-eus-upgrade

### CODE REVIEW

File: modules/updating-eus-to-eus-upgrade.adoc [hunk 1]
```diff
@@ -14,8 +14,8 @@ Following this procedure reduces the total update duration and the number of tim
 * Review the release notes for {product-title} 4.9 and 4.10
 * Review the release notes and product lifecycles for any layered products and OLM Operators. Some may require updates either before or during an EUS-to-EUS update.
 * Ensure that you are familiar with version-specific prerequisites, such as link:https://docs.openshift.com/container-platform/4.9/updating/updating-cluster-prepare.html#update-preparing-migrate_updating-cluster-prepare[administrator acknowledgement] that is required prior to updating from {product-title} 4.8 to 4.9.
-* Verify that your cluster is not running a version earlier than {product-title} 4.8.14.
-If your cluster is running a version than {product-title} 4.8.14, you must update to a later 4.8.z version prior to updating to 4.9.
+* Verify that your cluster is running {product-title} version 4.8.14 or later.
+If your cluster is running a version earlier than {product-title} 4.8.14, you must update to a later 4.8.z version prior to updating to 4.9.
```
Comment (tmalove) [inline modules/updating-eus-to-eus-upgrade.adoc:18]: Change to '...version **before** updating to 4.9'

**Review by jeana-redhat — APPROVED** [review]
LGTM

**Pull request merged**